### PR TITLE
Feat/level

### PR DESCRIPTION
**Pull request by kari0003**
Created basic arena, with baked navmesh

### CODE REVIEW

File: mono_crash.26f3a5e944.0.json [hunk 1]
```diff
@@ -0,0 +1,1801 @@
+{
```
Comment (balintant) [inline mono_crash.26f3a5e944.0.json]: CRASH!

Comment (kari0003) [reply mono_crash.26f3a5e944.0.json]: thats fair!

**Review by kocsislaci — APPROVED** [review]
Szep munka!

**Comment by kocsislaci** [comment]
I made a minor cleanup. Now I merge this branch. The spawn point tasks can be done later with now PR :)

**Pull request merged**